Search/Xapian.pm

```diff
@@ -46,7 +46,9 @@ sub EXEC_REALM {
 
 sub acquire_lock {
     my($proto, $req) = @_;
-    return $_M->new($req, 'Lock')->acquire_unless_exists(_lock_id($proto, $req));
+    $_M->new($req, 'Lock')->acquire_unless_exists(_lock_id($proto, $req));
+    unlink(File::Spec->catfile($_CFG->{db_path}, 'db_lock'));
```

Comment:
The unlink() was present in execute() previously. It seems like protection after a software crash which left a lock file. This might not be safe to unlink() on each call to acquire_lock()